### PR TITLE
match metadata to opencast standard

### PR DESCRIPTION
**Pull request by nataliemona**

### CODE REVIEW

File: functions/zoom-uploader.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def series_id_from_schedule(self):
             schedule = r['Item']
             logger.info(schedule)
 
-        zoom_time = self.created.astimezone(timezone(LOCAL_TIME_ZONE))
+        zoom_time = self.created_local
```
Comment (lbjay) [inline functions/zoom-uploader.py]: I feel like these time values are churning a lot?

File: functions/zoom-uploader.py [hunk 2]
```diff
@@ -268,6 +268,7 @@ def upload(self):
             logger.warning("Episode with mediapackage id {} already ingested"
                            .format(self.mediapackage_id))
             return None
+        self.get_series_catalog()
```
Comment (lbjay) [inline functions/zoom-uploader.py:267]: can it get to this point without a valid series id?

File: functions/zoom-uploader.py [hunk 3]
```diff
@@ -300,6 +301,18 @@ def create_mediapackage_id(self):
...
 
+    def get_series_catalog(self):
+
+        logger.info("Getting series catalog for series: {}"
+                    .format(self.opencast_series_id))
+
+        endpoint = "/series/{}.json".format(self.opencast_series_id)
+        resp = oc_api_request('GET', endpoint)
+
+        logger.debug(resp.text)
```
Comment (lbjay) [inline functions/zoom-uploader.py]: this should probably be something like, `logger.debug({'series_catalog': resp.text})`

File: functions/zoom-uploader.py (hunk 1 above)
Comment (nataliemona) [reply functions/zoom-uploader.py]: Yes, I agree...

File: functions/zoom-uploader.py (hunk 2 above)
Comment (nataliemona) [reply functions/zoom-uploader.py:267]: It's possible and then it would throw an error which is what we would want it to do.

**Review by lbjay — APPROVED** [review]

**Pull request merged**